.github/workflows/slow.yml

```diff
@@ -42,3 +43,10 @@ jobs:
       - name: Test with pytest
         run: |
           py.test -m slow
+          slack_api:
+      - name: Slack notify via api
```

Comment:
This name is incorrect.  We're not using the API, we're using the Incoming Webhook.